tests/test_help_pages.py

```diff
@@ -48,8 +42,11 @@ def getter(self: t.Any) -> None:
 @pytest.mark.help_page
 def test_access_help(no_api: None, subtests: SubTests) -> None:
     """Test, that all help screens are accessible without touching the api property."""
+    if parse_version(PULP_CLI_VERSION) < parse_version("0.24"):
+        pytest.skip("This test is incompatible with older cli versions.")
+
     runner = CliRunner()
-    for args in traverse_commands(main, []):
+    for args in traverse_commands(main.commands["deb"], ["deb"]):
```

Comment:
Not sure I understand this line. I guess I would need to look at the `main` function in `pulp_cli`.

Reply:
:lightbul: